frontend/src/components/TaskList.jsx

```
@@ -0,0 +1,144 @@
+    else {
+      await axiosInstance.put(`/tasks/${taskData._id}`, taskData);
+      setTasks(tasks.map((t) => (t._id === taskData._id ? taskData : t)));
+    }
+  };
+
+  return (
+    <div className={styles.container}>
+      <div className={styles.taskList}>
+        <div className={styles.taskHeader}>
+          <h2>Tasks</h2>
+          <button className={styles.addTaskButton} onClick={openNewTaskModal}>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)